test/object_test.rb

```diff
@@ -57,4 +58,19 @@ class ObjectTest < MiniTest::Spec
       _(source.album.songs[0].title).must_equal 1
     end
   end
-end
\ No newline at end of file
+
+  describe ".format_engine" do
```

Comment:
@dmuneras This is more like a test for using `Representable::Object` within decorator (as per the issue you've encountered). You can change the description here to mention that in order to not forget it later 😄 

For example,

```ruby
describe "renders object with decorator" do
end
```